diagnostic.js

```diff
@@ -15,7 +15,16 @@ db.dropDatabase();
 // House Targaryen, motto 'Fire and Blood'
 
 // your code begins here
-
+db.houses.insert([{
+  name: "Arryn",
+  motto: "As High as Honor",
+}, {
+  name: "Stark",
+  motto: "Winter is Coming",
+}, {
+  name: "Targaryen",
+  motto: "Fire and Blood",
+}]);
```

Comment:
this code wouldn't work because you are not inserting an Object. An Object is marked by `{ }`s. You are trying to insert an array.